src/components/contact.js

```diff
@@ -0,0 +1,31 @@
+export default function contact() {
+  const content = document.getElementById('content');
+  const contact = document.createElement('div');
```

Comment:
Make use of arrow functions instead of the `function` keyword. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/))